src/app/ReadHandler.h

```diff
@@ -349,7 +355,9 @@ class ReadHandler : public Messaging::ExchangeDelegate
 
     auto GetTransactionStartGeneration() const { return mTransactionStartGeneration; }
 
-    void UnblockUrgentEventDelivery();
+    /// @brief This allows the read handler to schedule a run as soon as the min interval has elapsed by setting the read handler to
+    /// a dirty state regardless of the attributes change, thus being reportable as soon as the min interval callback happens.
```

Comment:
```suggestion
    /// @brief Forces the read handler into a dirty state, regardless of what's going on with attributes.
    /// This can lead to scheduling of a reporting run immediately, if the min interval has been reached, 
    /// or after the min interval is reached if it has not yet been reached.
```